packages/common/src/models/response.ts

```diff
@@ -12,7 +12,7 @@ export class Response {
     /**
      * The headers of the response.
      */
-    headers?: { [key: string]: string };
+    private _headers: { [key: string]: string } = {};
```

Comment:
Are we sure we always want to have headers defined ?

Reply:
yes, else if we try to add one, it will fail.

Reply:
Should we do that only when we try to add one and it's undefined ? Because if there is none do we want the response to return an empty object as headers or nothing ?

Reply:
No the response to an http server will simply not contain any headers. And undefined for an object feels weird to me.